Cargo.toml

```diff
@@ -9,12 +9,13 @@ edition = "2021"
 name = "lib"
 path = "src/lib.rs"
 
-[features]
+[features] #REVIEW
 #TODO add import & lock feature
 default = ["arch"]
```

Comment:
Assuming this is meant for Arch Linux, is there a way to avoid mentioning it here (to make this file more generic)?

Reply:
Just default value for compiler. It can be changed from cargo during build time, but I think I may change this.